src/amo/components/AddonsCard/index.js

```diff
@@ -19,6 +19,7 @@ import './styles.scss';
 
 type DefaultProps = {|
   editing?: boolean,
+  isHomepageShelf?: boolean,
```

Comment:
As discussed in my reply to your question, I think this should be moved into `Props`.